api/preprints/serializers.py

```diff
@@ -76,6 +77,10 @@ class PreprintSerializer(JSONAPISerializer):
     is_preprint_orphan = ser.BooleanField(read_only=True)
     license_record = NodeLicenseSerializer(required=False, source='license')
 
+    reviews_state = ser.CharField(read_only=True, max_length=15)
+
+    submit_for_review = ser.BooleanField(required=False, write_only=True)
```

Comment:
Had a chat with Abram and suggested several changes. Basically, all workflow transitions should happen via your transition system and not implicitly. This means that when the user wants to submit an item for review (whether its a new preprint or going through another round), the UI should make a request for that to the transition workflow.

Also, we should not have another state for indicating that something should be seen by the public; `is_published` is good enough for that. `is_published = True` should mean that it's visible to unauthenticated users via the API and on SHARE. `is_published = False` should mean that it's not visible to unauthenticated users via the API nor on SHARE. Nothing else (once we divorce the nodes from the preprints) should be able to influence those items.